### PR TITLE
feat: use sqlite backend

### PR DESCRIPTION
**Pull request by samsja**
- [x] update code
- [x] update docs

### CODE REVIEW

**Review by JoanFM — CHANGES_REQUESTED** [review]
Remove the `if` condition from `index` request

**Review by JoanFM — CHANGES_REQUESTED** [review]
Maybe a `clear` endpoint can be useful

**Comment by samsja** [comment]
> 

done

**Comment by samsja** [comment]
>

**Comment by JoanFM** [comment]
> > Make sure to test that restarting the Flow keeps the `Indexer` with the data
> 
> Isn't it that test doing that ?
> 
> https://github.com/jina-ai/executor-simpleindexer/blob/895fb7076779aa244e5043fb3918d9cd507ca499/tests/test_simple_indexer.py#L80
> 
> 
> (well not exactly but I guess it is enough )

Make an integration test if possible

**Comment by samsja** [comment]
> Maybe a `clear` endpoint can be useful

done

**Review by JoanFM — CHANGES_REQUESTED** [review]

File: tests/test_simple_indexer.py [hunk 1]
```diff
@@ -210,3 +209,10 @@ def test_invalid_embedding_query(tmp_path, docs):
     indexer.index(DocumentArray([Document(), Document(embedding=np.array([1]))]))
     with pytest.raises(ValueError):
         indexer.search(DocumentArray([Document(embedding=np.array([1, 0]))]))
+
+def test_clear(tmp_path,docs):
+    metas = {'workspace': str(tmp_path / 'workspace')}
+    indexer = SimpleIndexer(metas=metas)
+    indexer.index(docs)
+    indexer.clear()
```
Comment (JoanFM) [inline tests/test_simple_indexer.py:217]: assert length before clear, to make sure it was not 0 before

**Review by JoanFM — CHANGES_REQUESTED** [review]

**Review by JoanFM — APPROVED** [review]

**Review by JoanFM — APPROVED** [review]

**Pull request merged**